appserver/ldapbp/src/main/java/com/sun/jndi/ldap/ctl/AuthorizationIDControl.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+ *
+ * Copyright (c) 2002-2018 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Copyright year should be 2018 in all the files since they are being introduced as new files.

Reply:
Corrected as per discussion.